### PR TITLE
Change all port 8080 to 8090 for test

### PR DESCRIPTION
**Pull request by wilsoniovlabs**
Change server config port to run the integration test and the rif relay server in parallel.

### CODE REVIEW

File: docs/launching_enveloping.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ In order to run an instance of Enveloping in Regtest:
     ```json5
     {
         url: 'localhost', // the interface where the relay server will be exposed
-        port: 8090, // the port where it will be running
+        port: 8095, // the port where it will be running
```
Comment (mortelli) [inline docs/launching_enveloping.md:30]: changes to this file should be undone, as these instructions are unrelated to the tests.

in fact, suggesting that the server should be launched in this port will cause the original problem to re-appear

File: docs/integration_guide.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ const deployVerifierAddress = '0x1938517B0762103d52590Ca21d459968c25c9E67'; // t
 const config = await resolveConfiguration(web3.currentProvider, {
     verbose: window.location.href.includes('verbose'),
     onlyPreferredRelays: true,
-    preferredRelays: ['http://localhost:8090'],
+    preferredRelays: ['http://localhost:8095'],
```
Comment (mortelli) [inline docs/integration_guide.md:54]: same applies to this file

File: docs/launching_enveloping.md [hunk 3]
```diff
@@ -89,6 +89,6 @@ In order to run an Enveloping instance in Testnet, clone the project then run th
 
 ## Troubleshooting
 
-#### Running some test and one of them throws: Error: listen EADDRINUSE: address already in use :::8090
+#### Running some test and one of them throws: Error: listen EADDRINUSE: address already in use :::8095
```
Comment (mortelli) [inline docs/launching_enveloping.md:92]: we can leave this one

File: docs/integration_guide.md (hunk 2 above)
Comment (wilsoniovlabs) [reply docs/integration_guide.md]: Fixed.

File: docs/launching_enveloping.md (hunk 1 above)
Comment (wilsoniovlabs) [reply docs/launching_enveloping.md]: Fixed.

Comment (mortelli) [reply docs/launching_enveloping.md]: please apply the same correction to the rest of the file

File: docs/integration_guide.md [hunk 4]
```diff
@@ -185,7 +185,7 @@ const partialConfig: Partial<EnvelopingConfig> = {
     chainId: chainId,
     relayVerifierAddress: relayVerifier.address, // The verifier that will verify the relayed transaction
     deployVerifierAddress: deployVerifier.address, // The verifier that will verify the smart wallet deployment
-    preferredRelays: ['http://localhost:8090'] //If there is a preferred relay server.
+    preferredRelays: ['http://localhost:8095'] //If there is a preferred relay server.
```
Comment (mortelli) [inline docs/integration_guide.md:188]: here

File: docs/launching_enveloping.md [hunk 5]
```diff
@@ -40,7 +40,7 @@ In order to run an instance of Enveloping in Regtest:
     }
     ```
 3. From the root directory run `node dist/src/cli/commands/enveloping.js relayer-run --config jsrelay/config/relay-config.json`.
-4. To check if it is working, run `curl http://localhost:8090/getaddr`.
+4. To check if it is working, run `curl http://localhost:8095/getaddr`.
```
Comment (mortelli) [inline docs/launching_enveloping.md:43]: here

File: docs/launching_enveloping.md [hunk 6]
```diff
@@ -66,8 +66,8 @@ To implement and use your own replenish strategy:
...
-3. In another terminal run `curl http://localhost:8090/getaddr` which will return some JSON with information of the running jsRelay Server, for example:
+2. In a terminal run `node dist/src/cli/commands/enveloping.js relayer-run --rskNodeUrl "http://localhost:4444" --relayHubAddress=<RELAY_HUB_CONTRACT_ADDRESS> --deployVerifierAddress=<DEPLOY_VERIFIER_CONTRACT_ADDRESS> --relayVerifierAddress=<RELAY_VERIFIER_CONTRACT_ADDRESS> --versionRegistryAddress=<VERSION_REGISTRY_CONTRACT_ADDRESS> --url <RELAY_URL> --port 8095 --workdir enveloping_relay --checkInterval 30000` where `<RELAY_HUB_CONTRACT_ADDRESS>` is the address for the relayHub you are using in the current network [(see Testnet Contracts section)](#c02.1), `<RELAY_URL>` in most cases will be `http://localhost`, and the server will be reachable in `<RELAY_URL>:port` unless `<RELAY_URL>` already defines a port (e.g, if `<RELAY_URL>` is `http://localhost:8095/jsrelay`)
```
Comment (mortelli) [inline docs/launching_enveloping.md:69]: here

File: docs/launching_enveloping.md [hunk 7]
```diff
@@ -66,8 +66,8 @@ To implement and use your own replenish strategy:
...
+3. In another terminal run `curl http://localhost:8095/getaddr` which will return some JSON with information of the running jsRelay Server, for example:
```
Comment (mortelli) [inline docs/launching_enveloping.md:70]: here

**Review by mortelli — COMMENTED** [review]
please undo the following changes for the reasons laid out in my previous review.

thanks

File: test/TestUtils.ts [hunk 8]
```diff
@@ -45,12 +45,13 @@ import {
...
 //@ts-ignore
 import sourceMapSupport from 'source-map-support';
+import { RIF_RELAY_HOST, RIF_RELAY_PORT } from './Utils';
 //@ts-ignore
 sourceMapSupport.install({ errorFormatterForce: true });
 
 const RelayHub = artifacts.require('RelayHub');
 
-const localhostOne = 'http://localhost:8095';
+const localhostOne = `${RIF_RELAY_HOST}:${RIF_RELAY_PORT}`;
```
Comment (antomor) [inline test/TestUtils.ts:54]: could we use directly the variable exported at `test/TestUtils.ts#53`?

File: test/Utils.ts [hunk 9]
```diff
@@ -5,6 +5,9 @@ import path from 'path';
 import { DeploymentResult } from './TestSetup';
 import { RelayHubConfiguration } from '@rsksmart/rif-relay-common';
 
+export const RIF_RELAY_HOST = 'http://localhost';
+export const RIF_RELAY_PORT = '8095';
```
Comment (antomor) [inline test/Utils.ts:9]: I would also expose a `RIF_RELAY_URL` constant to be used when we need both

Comment (antomor) [reply test/Utils.ts:9]: Could we read these parameters from the config file?

File: test/relayserver/ServerTestEnvironment.ts [hunk 10]
```diff
@@ -74,7 +75,7 @@ abiDecoder.addABI(TestVerifierEverythingAccepted.abi);
 // @ts-ignore
 abiDecoder.addABI(TestDeployVerifierEverythingAccepted.abi);
 
-export const LocalhostOne = 'http://localhost:8095';
+export const LocalhostOne = `${RIF_RELAY_HOST}:${RIF_RELAY_PORT}`;
```
Comment (antomor) [inline test/relayserver/ServerTestEnvironment.ts:78]: maybe also this constant can be removed to use the one declared in `test/Utils.ts` or if you prefer, we could move all the constants here, or alternatively, we could define a `test/constants.ts` file

File: test/Utils.ts (hunk 9 above)
Comment (wilsoniovlabs) [reply test/Utils.ts]: Good catch.
